src/pages/Root/Root.js

```diff
@@ -91,20 +97,20 @@ const Root = ({
 
 	return (
 		<ThemeProvider theme={themeStyle}>
-			<div>
-				<Router>
-					<RootContainer>
+			<Router history={history}>
+				<RootContainer>
+					<MainLayout>
+						<Header />
+						<WalletPopup />
+						<GweiPopup />
 						<Switch>
-							<Route path="/trade">
-								<Trade />
-							</Route>
-							<Route path="/">
-								<Trade />
-							</Route>
+							<Route path={ROUTES.Trade} component={Trade} />
+							<Route path={ROUTES.Loans} component={Loans} />
+							<Redirect to={ROUTES.Trade} />
```

Comment:
When users land on the page they will be redirected to `/trade`. Let me know if its OK (this helps with `NavLink` detect that we are on the exchange).

Reply:
yeah I'm happy with that!